src/core/algorithms/gfd/graph_descriptor.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <map>
+#include <string>
+
+#include <boost/graph/adjacency_list.hpp>
+
+struct vertex {

```

Comment:
for struct and class names, we also use `PascalCase` (https://google.github.io/styleguide/cppguide.html#Type_Names)